src/explicitode.jl

```diff
@@ -350,216 +411,89 @@ tv, xv, polynv = taylorinteg(f!, [3, 3], 0.0, 0.3, 25, 1.0e-20, maxsteps=100, Va
 ```
 
 """
-function taylorinteg(f, x0::U, t0::T, tmax::T, order::Int, abstol::T,
-    params = nothing; maxsteps::Int=500, parse_eqs::Bool=true) where {T<:Real, U<:Number}
-
-    # Allocation
-    tv = Array{T}(undef, maxsteps+1)
-    xv = Array{U}(undef, maxsteps+1)
-
-    # Initialize the Taylor1 expansions
-    t = Taylor1( T, order )
-    x = Taylor1( x0, order )
-
-    # Initial conditions
-    nsteps = 1
-    @inbounds t[0] = t0
-    @inbounds tv[1] = t0
-    @inbounds xv[1] = x0
-    sign_tstep = copysign(1, tmax-t0)
-
-    # Determine if specialized jetcoeffs! method exists
-    parse_eqs = _determine_parsing!(parse_eqs, f, t, x, params)
-
-    # Integration
-    while sign_tstep*t0 < sign_tstep*tmax
-        δt = taylorstep!(f, t, x, abstol, params, parse_eqs) # δt is positive!
-        # Below, δt has the proper sign according to the direction of the integration
-        δt = sign_tstep * min(δt, sign_tstep*(tmax-t0))
-        x0 = evaluate(x, δt) # new initial condition
-        @inbounds x[0] = x0
-        t0 += δt
-        @inbounds t[0] = t0
-        nsteps += 1
-        @inbounds tv[nsteps] = t0
-        @inbounds xv[nsteps] = x0
-        if nsteps > maxsteps
-            @warn("""
-            Maximum number of integration steps reached; exiting.
-            """)
-            break
-        end
-    end
-
-    #return tv, xv
-    return view(tv,1:nsteps), view(xv,1:nsteps)
+function taylorinteg(f, x0::U, t0::T, tmax::T, order::Int, abstol::T, params = nothing;
+    maxsteps::Int=500, parse_eqs::Bool=true) where {T<:Real, U<:Number}
+    return taylorinteg(f, x0, t0, tmax, order, abstol, Val(false), params; maxsteps, parse_eqs)
```

Comment:
I think the errors that appear when testing with Julia v1.0 (for example [here](https://github.com/PerezHz/TaylorIntegration.jl/runs/5821366447?check_suite_focus=true#step:6:27)) are due to the fact that you must include the keyword explicitly for the keyword arguments of a function. Can you test this?